src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs

```diff
@@ -54,7 +54,10 @@ protected WKWebView PrepareWKWebView()
         {
             WKWebViewConfiguration wkconfg = new WKWebViewConfiguration() { };
 #if !IS_APPCENTER_BUILD
```

Comment:
why keeping this?

Reply:
idk. i removed it in the other pr (link somewhere here)

---
In reply to: [378475327](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1627#discussion_r378475327) [](ancestors = 378475327)